FiveCalls/FiveCalls/AppState.swift

```diff
@@ -13,6 +13,16 @@ import os
 class AppState: ObservableObject, ReduxState {
     @Published var globalCallCount: Int = 0
     @Published var issueCallCounts: [Int: Int] = [:]
+    // issueCompletion is a local cache of completed calls: an array of contact ids keyed by an issue id
+    @Published var issueCompletion: [Int: [String]] = [:] {
+        didSet {
+            // NSNumber (bridged automatically from Int) is not supported as a key in a plist dictionary, so we stringify and unstringify
```

Comment:
Does this also mean we couldn't leverage `@AppStorage` here?

Reply:
Not that I can see, I can't get away from "no exact matches in call to initializer" if I use anything except a bare supported plist type (i.e. no arrays or dicts) for @AppStorage.